lib/verdeps/find_package_import_comment.go

```diff
@@ -23,16 +23,24 @@ func findPackageImportComment(
 ) (fromIndex int, toIndex int) {
 	// Read until the end of the line or the end of the file.
 	packageEndIndex := packageStartIndex
-	for packageEndIndex < len(fileData) && fileData[packageEndIndex] != '\n' {
+	for packageEndIndex >= 0 &&
```

Comment:
nit: I don't think these >= checks should be in here when you're only adding. I know it's symmetrical but it's confusing why you would check it imo